packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -159,6 +191,10 @@ class RenderListWheelViewport
       'Cannot renderChildrenOutsideViewport and clipToSize since children '
       'rendered outside will be clipped anyway.';
 
+  /// The delegate that manages the children of this object.
+  ListWheelChildManager get childManager => _childManager;
```

Comment:
Everything that can be set via the constructor must be settable via a setter.

Reply:
fixed.